assets/Languages/zh-HK.xlf

```diff
@@ -13,18 +12,18 @@
 				</trans-unit>
 				<trans-unit id="contributors">
 					<source>Unknown</source>
-					<target>Alvin Wong, I-Circle BVE (http://bve.i-circle.net)</target>
+					<target>Alvin Wong@I-Circle BVE, Kenny Hui@HKTSS</target>
```

Comment:
I would like to point out that, the very original translation was attributed to "I-Circle BVE (http://bve.i-circle.net)" and I (Alvin Wong) later translated some additional strings, but we are two separate entities. It is not supposed to be "Alvin Wong@I-Circle BVE".